django/db/backends/base/base.py

```diff
@@ -182,6 +182,14 @@ def queries(self):
                 "will be returned.".format(self.queries_log.maxlen))
         return list(self.queries_log)
 
+    def get_database_version(self):
+        raise NotImplementedError('subclasses of BaseDatabaseWrapper may require a get_database_version() method')
```

Comment:
```suggestion
        raise NotImplementedError('subclasses of BaseDatabaseWrapper may require a get_database_version() method.')
```
We can add tests for this exception in the `tests.backends.base.test_base.DatabaseWrapperTests`.